### ISSUE TITLE
Non-cantrip spells don't show up for my PC's when using Action Hud

### ISSUE THREAD

**Issue by mvanduijvenvoorde** [espositos/fvtt-tokenactionhud#69]
Non-cantrip spells are not selectable when using Token Action Hud, just the cantrip spells, for their characters. They DO show up when I'm using NPC's.

First I thought it had to do with one of my PCs not having spell slots (they use Spell Points), but the other PC does have spell slots and also can't click level 1 or 2 spells.

Any idea what's wrong?

VTT: FoundryVTT (0.6.6)
System: 5e (0.96)
Installed Modules:

MinorQol
Token Magic FX
MaxCrit
Turn Alert
Custom Hotbar
Sky's Alternate Char Sheet
Target Enhancements
Easy Target
Health Estimate 
Permission Viewer
Dice So Nice
Better NPC sheet

**Comment by espositos** [comment]
I'm not sure, I'll take a look into it. Are you available on discord if I have questions?

**Comment by espositos** [comment]
And are those spells marked as prepared?

**Comment by espositos** [comment]
I can't replicate this. I guess it is because the character has not marked the spells as prepared. If you could check this and let me know. Feel free to get in touch with me on Discord if you still have problems (^ and stick#0520).

**Comment by mvanduijvenvoorde** [comment]
The spells weren't "Prepared" so I clicked them as Prepared now. This fixes it for the character that uses Spell Slots! However the character that uses Spell Points, and has Spell Slots at 0 for each level, does not get the Spell options in the menu. If I set one of the level with available Spell Slots (so like 2/2, instead of 0/2), ALL of the spell options become visible, even the ones were the character still has available Spell Slots at 0/2.
